dle/.env.example

```diff
@@ -1,5 +1,14 @@
 # General
 # ------------------------------------------------------------------------------
+
+API_ENDPOINT="searchrx-1695064113.us-east-1.elb.amazonaws.com"
+SEARCHRX_URL="searchrx-1695064113.us-east-1.elb.amazonaws.com"
```

Comment:
Might be better to leave the other ones as the default uncommented ones since people doing local deployment will probably want to use `.env.example` without altering.